routing/routing.go

```diff
@@ -0,0 +1,97 @@
+package routing
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"os/exec"
+	"text/template"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	eth1FileName = "/etc/systemd/network/10-eth1.network"
+)
+
+type Params struct {
+	ENIAddress    string
+	ENIGateway    string
+	ENISubnetSize int
+}
+
+func ConfigureNetworkRoutingForENI(eniIP string, eniSubnet *net.IPNet) error {
+
+	params := Params{
+		ENIAddress:    eniIP,
+		ENIGateway:    eniGateway(eniSubnet),
+		ENISubnetSize: eniSubnetSize(eniSubnet),
+	}
+
+	err := renderRoutingNetworkdFile(params)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = restartNetworkd()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	return nil
+}
+
+func renderRoutingNetworkdFile(params Params) error {
+	var buff bytes.Buffer
+	t := template.Must(template.New("routing").Parse(networkRoutingTemplate))
+
+	err := t.Execute(&buff, params)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = ioutil.WriteFile(eth1FileName, buff.Bytes(), os.ModeAppend)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
+func restartNetworkd() error {
+	cmdReload := exec.Command("/usr/bin/systemctl", "daemon-reload")
+	err := cmdReload.Run()
+	if err != nil {
+		return microerror.Maskf(err, fmt.Sprintf("failed to reload daemon for systemd: %s", err))
```

Comment:
This won't print the command output. I'd suggest calling `.Output` instead or `.Run` and print the output here in case of error.

Reply:
done